README.md

```diff
@@ -100,7 +108,7 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+A schema is a file that establishes parameters for tables in a database.  A one-to-many relationship in a SQL database is represented by relationships between tables.  For instance, you may have a table of customers and a table of orders, each customer can have many orders.
```

Comment:
This isn't wrong, but I was hoping to see a mention of foreign keys, since that's how we specifically represent a relationship in sql, e.g. an order may have a `person_id` column.